lncfg/address.go

```diff
@@ -18,6 +18,8 @@ var (
 	loopBackAddrs = []string{"localhost", "127.0.0.1", "[::1]"}
 )
 
+const allInterfacesIP = "0.0.0.0"
```

Comment:
In this case, we want to use: [`IsUnspecified`](https://pkg.go.dev/net#IP.IsUnspecified)